### PR TITLE
Work on descriptions widget conditionally rendering

### PR DESCRIPTION
**Pull request by cmonez**
Create test spec based on this functionality. @andyoe0102 @samasastudio @krisdo

### CODE REVIEW

File: client/Index.jsx [hunk 1]
```diff
@@ -4,24 +4,27 @@ import MainDescription from './components/MainDescription.jsx';
...
   componentDidMount() {
     axios.get('/api/description/0').then(({ data }) => {
-      this.setState({ description: data }).catch((error) => {
-        console.log('Error fetching data', error);
+      this.setState({ descriptions: data.description }, () => {
+        console.log('Updated state,', this.state);
       });
     });
+    // });
   }
```
Comment (samasastudio) [inline client/Index.jsx:17]: dont forget to handle errors in the catch

File: client/components/MainDescription.jsx [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+    super(props);
+    this.state = {
+      showMore: false,
+    };
+    // this.showMore = this.showMore.bind(this);
+  }
+
+  showMore(event) {
+    this.setState({ showMore: true });
+  }
```
Comment (samasastudio) [inline client/components/MainDescription.jsx:14]: should showMore be a method be more modular in the sense that it just changes showMore to the opposite of its current state?  that way you can use it to hide the "more" observations as well?  trying to remember if the site even does that and can't find it on my example page.

File: client/Index.jsx (hunk 1 above)
Comment (cmonez) [reply client/Index.jsx:17]: Ugh. I had the catch there and deleted it! Thanks for catching it.

File: client/components/MainDescription.jsx (hunk 2 above)
Comment (cmonez) [reply client/components/MainDescription.jsx:14]: The way its implemented on Airbnb is that once you click "read more" you cannot then hide - no "read less" functionality exists. It's a one time thing. Thats a good question though!

**Pull request merged**